### DISCUSSION TITLE
Bold or italic words within curriculum vitae R markdown hindman::vitae

### DISCUSSION THREAD

**Discussion by chitrabaniya** [community/community#52258] [Programming Help]
### Select Topic Area

Question

### Body

Normall ways of R Markdown pdf bold or  italic was not happen in hindman::vitae, please help.

**Comment by SMelidoni** [comment]
Hi @chitrabaniya 👋🏽

The `vitae` package uses the `pandoc` template system to generate CVs, which may cause some issues with traditional R Markdown formatting. However, you can still apply bold and italic formatting using LaTeX syntax directly in your R Markdown file.

1. **Bold text**: Use `\textbf{}` to make the text inside the curly braces bold.

Example: `This is a \textbf{bold} text.`

2. **Italic text**: Use `\textit{}` to make the text inside the curly braces italic.

Example: `This is an \textit{italic} text.`

By using these LaTeX commands in your R Markdown file, you should be able to achieve bold and italic formatting when using the `vitae` package to generate your CV.

Hope this helps!

**Reply by chitrabaniya** [reply]
Hello dear SMelidoni,
Thanks for your reply. 
All my entries for example list of publications, thesis supervision etc are inside R-chunk. This is what I have been used ti Hyndman's vitae package in R.
Thus I am stucked to get bold or italic.
 Your suggest good when I write outside of the R-Code chunk.
Thanks.

**Reply by SMelidoni** [reply]
Hi,

I see what you mean, maybe you can achieve this by using the `paste()` or `sprintf()` functions to concatenate strings with the LaTeX formatting commands.

Here's how to do it:

1. **Bold text within R code chunk**: Use the `paste()` or `sprintf()` function with the `\textbf{}` LaTeX command.

   Example:

```r
   bold_text <- "bold"
   cat(paste("This is a", paste0("\\textbf{", bold_text, "}"), "text."))

   or

   bold_text <- "bold"
   cat(sprintf("This is a \\textbf{%s} text.", bold_text))
```

2. **Italic text within R code chunk**: Use the `paste()` or `sprintf()` function with the `\textit{}` LaTeX command.

    Example:

```r
   italic_text <- "italic"
   cat(paste("This is an", paste0("\\textit{", italic_text, "}"), "text."))

   or

   italic_text <- "italic"
   cat(sprintf("This is an \\textit{%s} text.", italic_text))
```

Hopefully by using these techniques you should be able to include bold and italic formatting within R code chunks in your R Markdown document while using the `vitae` package.

**Reply by chitrabaniya** [reply]
Dear  Simone Melidoni, 
Thank you for your help.
I could not incorporate the codes that you gave. Here is what I got stacked with. 
It is one or two words within inside tribble(........................). For example, let us say I want to make Traditional in the below as italics. How can I do it? 
```{r}
tribble(
  ~Year, ~Type, ~Desc, 
'2001', "Bijaya (Adhikari) Aryal", "*Traditional* veterinary medicine practised in area adjoining Panchase forest of Prabat, Nepal."
) %>% 
  brief_entries(
    glue::glue("{Type}"),
    Year, 
    Desc
  )
```

**Comment by SMelidoni** [comment]
Hi @chitrabaniya,
 
I think I understand your situation now. In this case, you can directly include the LaTeX commands within the text strings inside the tribble() function. 

Here's how you can make "Traditional" italic:

```
tribble(
  ~Year, ~Type, ~Desc, 
'2001', "Bijaya (Adhikari) Aryal", "\\textit{Traditional} veterinary medicine practised in area adjoining Panchase forest of Prabat, Nepal."
) %>% 
  brief_entries(
    glue::glue("{Type}"),
    Year, 
    Desc
  )
```

In this example, I replaced *Traditional* with \\textit{Traditional}. This will render "Traditional" in italics in the output document. Note the use of double backslashes \\ to escape the backslash for the LaTeX command.

I hope this works! If you have any further questions or need assistance, please let me know.

**Reply by chitrabaniya** [reply]
Dear Simone Melidoni,
Thank you a lot for your help. All mentined tries were negatve for me. That is strange.  I can not figure out.

**Reply by c-martinez** [reply]
Hi @chitrabaniya, I new to vitae. I got the solution proposed by @SMelidoni to work, by adding `.protect = FALSE`

```
tribble(
  ~Year, ~Type, ~Desc, 
'2001', "Bijaya (Adhikari) Aryal", "\\textit{Traditional} veterinary medicine practised in area adjoining Panchase forest of Prabat, Nepal."
) %>% 
  brief_entries(
    glue::glue("{Type}"),
    Year, 
    Desc, 
    .protect = FALSE
  )
```

![image](https://github.com/community/community/assets/7782231/b248eb95-9486-40a5-b0dd-13e693bafb7c)

(I am using the latexcv style)

I hope this helps?

**Comment by chitrabaniya** [comment]
Hi,
I am waiting help if there are any on my above problem.

**Reply by chitrabaniya** [reply]
Hello Everyone,  nothing yet  happened.
